Procfile

```diff
@@ -0,0 +1,2 @@
+worker: bundle exec sidekiq -C config/sidekiq.yml
```

Comment:
NB: There aren't currently any jobs for the worker to run and I'm not actually sure *when* we'll need Sidekiq.

Reply:
Ooooh. Is it worth keeping this just in case?